CMakeLists.txt

```diff
@@ -25,6 +25,16 @@ cmake_minimum_required(VERSION 3.1.3)
 project(LibTaskForce LANGUAGES CXX)
 set(CMAKE_CXX_EXTENSIONS False CACHE BOOL "Enable/Disable compiler-specific C++ extensions")
 set(CMAKE_CXX_STANDARD 14 CACHE STRING "Which C++ standard to use")
+set(LIBTASKFORCE_BUILD_SHARED FALSE CACHE BOOL "Build a shared library, rather than a static library")
+set(LIBTASKFORCE_BUILD_PIC FALSE CACHE BOOL "Build the static libraric with PIC enabled")
```

Comment:
This one I did know about. I suppose it's nitpicky, but I wanted to set PIC only for the library and not the test executables. CMAKE_POSITION_INDEPENDENT_CODE sets it for all libraries and executables. But it doesn't harm anything if you'd like me to change it back